scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerImages.java

```diff
@@ -70,6 +70,9 @@
     @Source("images/job_kill_16.png")
     ImageResource job_kill_16();
 
+    @Source("images/job_export_32.png")
+    ImageResource scheduler_export_32();
+
     @Source("images/job_output_16.png")
     ImageResource job_output_16();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)